src/fragments/lib/auth/common/mfa_preferences/common.mdx

```diff
@@ -0,0 +1,37 @@
+<Callout>
+Fetching and Updating MFA preferences is only available through the escape hatch APIs under the AWS Cognito Auth plugin. 
+</Callout>
+
+## Fetch the current user's MFA preferences
+
+Invoke the following api to get the MFA preferences for the user.
```

Comment:
```suggestion
Invoke the following API to get the MFA preference for the current user.
```

I would also change `get` to `fetch` or `retrieve`

Reply:
Updated.